downstream/modules/platform/con-operator-upgrade-prereq.adoc

```diff
@@ -3,9 +3,8 @@
 = Prerequisites
 
 
-[role="_abstract"]
-To upgrade to a newer version of {OperatorPlatform}, it is recommended that you do the following:
+To upgrade to a newer version of {OperatorPlatformNameShort}, you must:
 
-* Create AutomationControllerBackup and AutomationHubBackup objects. For help with this see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/red_hat_ansible_automation_platform_operator_backup_and_recovery_guide/index#aap-backup-recommendations[Creating Red Hat Ansible Automation Platform backup resources]
-//See (Backup and Restore) for information on creating backup objects. [add link to new backup and restore doc when complete]
-* Review the release notes for the new {PlatformNameShort} version to which you are upgrading and any intermediate versions.
+* Create AutomationControllerBackup and AutomationHubBackup objects. For help with this see link:{URLOperatorBackup}[{TitleOperatorBackup}]
+* Review the link:{URLReleaseNotes}[{TitleReleaseNotes}] for the new {PlatformNameShort} version to which you are upgrading and any intermediate versions.
+* Determine the type of upgrade you want to perform. See the Channel Upgrades section for more information.
```

Comment:
Should Channel Upgrades be an xref or link?

Reply:
Xref will adjust!